pkg/controller/clusterdeployment/clusterdeployment_deleted.go

```diff
@@ -42,27 +42,32 @@ func (r *ReconcileClusterDeployment) handleDelete(request reconcile.Request, ins
 		BaseDomain: instance.Spec.BaseDomain,
 	}
 	err := pdData.ParsePDConfig(r.client)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
+	// if there is an error, no configuration (configmap or secret) was found.
```

Comment:
Shouldn't we parse the error to actually confirm this?

Reply:
+1, it would be better to use the `IsNotFound(err)` method from `"k8s.io/apimachinery/pkg/api/errors"`

Reply:
+1